assets/js/components/legacy-setup/wizard-step-authentication.js

```diff
@@ -33,10 +33,48 @@ import { Component } from '@wordpress/element';
 import { Button } from 'googlesitekit-components';
 import { Cell, Grid, Row } from '../../material-components';
 import OptIn from '../OptIn';
+import { VIEW_CONTEXT_SPLASH } from '../../googlesitekit/constants';
+import { setItem } from '../../googlesitekit/api/cache';
+import { trackEvent } from '../../util';
 
 class WizardStepAuthentication extends Component {
+	constructor( props ) {
+		super( props );
+		this.onButtonClick = this.onButtonClick.bind( this );
+	}
+
+	async onButtonClick() {
+		const { connectURL, isSiteKitConnected } = this.props;
+
+		await Promise.all( [
+			// Cache the start of the user setup journey.
+			// This will be used for event tracking logic after successful setup.
+			setItem( 'start_user_setup', true ),
+			trackEvent(
+				VIEW_CONTEXT_SPLASH,
```

Comment:
I just noticed that we're using a different category for the start events than the complete. I'll fix this to use the same for consistency.